client/cypress/utils/e2e.ts

```diff
@@ -47,3 +47,10 @@ export const connectWallet = (
   cy.switchToMetamaskNotification();
   return cy.acceptMetamaskAccess();
 };
+
+export const checkProjectsViewLoaded = (): Chainable<any> => {
+  cy.get('[data-test=ProjectsView__ProjectsList]')
```

Comment:
For archived projects it should be `ProjectsView__ProjectsList--archive`

Reply:
Good catch, done.